samples/VirtoCommerce.Exp.ExtensionSamples/VirtoCommerce.Exp.ExtensionSamples.csproj

```diff
@@ -14,7 +14,7 @@
         <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="8.0.1" />
         <PackageReference Include="MediatR" Version="8.0.1" />
         <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.8" />
-        <PackageReference Include="VirtoCommerce.Platform.Core" Version="3.57.0" />
+        <PackageReference Include="VirtoCommerce.Platform.Core" Version="3.66.0" />
```

Comment:
Update `<PackageReference Include="VirtoCommerce.Platform.Security" Version="3.57.0" />` to `<PackageReference Include="VirtoCommerce.Platform.Security" Version="3.66.0" />`.